bindings/c/CMakeLists.txt

```diff
@@ -337,7 +337,7 @@ if(NOT WIN32)
     COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
         --build-dir ${CMAKE_BINARY_DIR}
         --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-        --upgrade-path "7.2.0" "7.3.0" "7.2.0"
+        --upgrade-path "71.2.0" "71.3.0" "71.2.0"
```

Comment:
Can we also do this for fdb_c_upgrade_to_future_version_blob_granules just below too? There is a PR in the stage branch that fixes that test and enables it